### ISSUE TITLE
block-scoped-var false positive on export .. from .. construction

### ISSUE THREAD

**Issue by DenisIzmaylov** [eslint/eslint#2807]
Example:
```javascript
export createRedux from './createRedux';
...
```
Which generates follow output:
```
src/index.js
  2:7  error  "createRedux" used outside of binding context         block-scoped-var
...
```

**Comment by nzakas** [comment]
This syntax is actually invalid and is a proposed addition to ES7:
syntax, but that is part of the experimental https://github.com/leebyron/ecmascript-more-export-from

Really, the parser should throw an error here.

**Comment by nzakas** [comment]
Are you using the default parser or babel-eslint?

**Comment by gyandeeps** [comment]
Closing due to no response.
Feel free to reopen if the issue persists.
